XRTK-Core/Packages/com.xrtk.core/Definitions/Platforms/CurrentBuildTargetPlatform.cs

```diff
@@ -22,8 +22,6 @@ public sealed class CurrentBuildTargetPlatform : BasePlatform
         /// <returns>True, if any build target is active.</returns>
         public static bool IsBuildTargetActive(List<IMixedRealityPlatform> platforms)
         {
-            if (!Application.isEditor) { return false; }
-
             var isEditor = false;
```

Comment:
Almost. There is another "isEditor" validation below. Let's remove that as well for completeness.

Reply:
That's a different check